lib/fathead/arrow_docs/parse.py

```diff
@@ -37,8 +37,11 @@ def yield_api_example(api_examples):
 
         abstract = clean_output(api_example.dd.p.text)
         abstract = abstract_fmt.format(abstract, code)
+        
+        # redirect_alias: => 'arrow.arrow.Arrow.clone' => arrow clone
+        redirect_alias = 'arrow ' + title.split('.')[-1]
 
-        yield [title, 'A', '', '', '', '', '', '', '', '', '', abstract, url]
```

Comment:
@danidee10 the article entry must be in the `output.txt` for the article to exist at all.

Redirect in the `output.txt` are only a way to create aliases for existing articles in the output.txt

Using redirects.txt may be a simpler approach but the idea is that redirects.txt are written manually.

If we can programatically generate redirects and add them to the output.txt that is ideal 👍 


-----

### Example

`output.txt`
```
arrow.arrow.Arrow.clone	A	....more content...
Arrow.clone	R	arrow.arrow.Arrow.clone
Arrow clone	R	arrow.arrow.Arrow.clone
....
```

With only the first line in the output the query "arrow.arrow.Arrow.clone" is the only thing that would return our result.
With the Redirect entires, the new aliases "Arrow.clone" and "Arrow clone" will also return our "arrow.arrow.Arrow.clone" result when queried